src/native/fs/fs_napi.cpp

```diff
@@ -1418,6 +1434,12 @@ struct GetPwName : public FSWorker
         set_getpwnam_res(env, res, *_getpwnam_res);
         _deferred.Resolve(res);
     }
+
+    ~GetPwName()
+    {
+        free(buf); 
```

Comment:
for proper safety this should be:
```
if (buf) { free(buf); buf = NULL; } 
```
but in cpp we should prefer unique_ptr for managing such allocations with less error prone code.

Reply:
removed because I used unique_ptr